### ISSUE TITLE
Code Coverage for RuntimeIdentifiers

### ISSUE THREAD

**Issue by nagilson** [dotnet/sdk#28048]
Based on https://github.com/dotnet/sdk/issues/27985 which was resolved by https://github.com/dotnet/sdk/pull/28006, it would be helpful if we had a regression test with projects that leverage RuntimeIdentifiers but not a RuntimeIdentifier. In addition, we may be able to provide a better patch in the long-term now that the other builds are presumably unblocked. In discussion with @dsplaisted, we should likely move it to another task/target.

**Comment by nagilson** [comment]
Huzzah, this can be closed
